Chaincase.UI/Services/Extensions.cs

```diff
@@ -24,5 +27,10 @@ public static IServiceCollection AddUIServices(this IServiceCollection services)
 
 			return services;
 		}
+
+		public static async Task NavigateBack(this NavigationManager navigationManager, IJSRuntime runtime, string route)
```

Comment:
why pass `IJSRuntime runtime` ?

Reply:
How does this really differ from `NavigateTo`?

Reply:
You'll want this ready for when we re-add transitions, which signals to the transition engine to do a different transition